packages/oslo-validator-jsonld/lib/JsonldValidationService.ts

```diff
@@ -45,41 +45,41 @@ export class JsonldValidationService implements IService {
 
     if (resultUris.isValid) {
       this.logger.info(
-        'Validation successful! All assigned URIs are whitelisted.',
+        'Validation successful! All assigned URIs are whitelisted.'
       );
     } else {
       this.logger.info(
-        `Validation found ${resultUris.invalidEntries.length} non-whitelisted assigned URIs`,
+        `Validation found ${resultUris.invalidEntries.length} non-whitelisted assigned URIs`
       );
     }
 
     if (resultSentences.isValid) {
       this.logger.info(
-        'Validation successful! All sentences seem to be valid, no spelling mistakes found.',
+        'Validation successful! All sentences seem to be valid, no spelling mistakes found.'
       );
     } else {
       this.logger.info(
-        `Validation found ${resultSentences.invalidEntries.length} sentences with spelling mistakes.`,
+        `Validation found ${resultSentences.invalidEntries.length} sentences with spelling mistakes or missing definition(s).`
```

Comment:
I tried to keep validators 'simple': one task per piece. Here we are mixing them (spelling with definitions).